xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserPreferencesSheet.xml

```diff
@@ -340,8 +340,8 @@
       &lt;div class="passwordManagement"&gt;
         &lt;h2&gt;$escapetool.xml($services.localization.render('platform.core.profile.section.security'))&lt;/h2&gt;
         &lt;span class="buttonwrapper"&gt;
-          &lt;a id="changePassword" href="$doc.getURL('view', 'xpage=passwd')"&gt;$escapetool.xml(
-            $services.localization.render('platform.core.profile.changePassword'))&lt;/a&gt;
+          &lt;button id="changePassword" onclick="location='$doc.getURL('view', 'xpage=passwd')'"&gt;$escapetool.xml(
```

Comment:
Hmmmm it means it won't work anymore when javascript is disabled. Not a big fan of this onclick attribute to be honest.

Reply:
Reverted those changes in https://github.com/xwiki/xwiki-platform/pull/3148/commits/d0434709df1df8a87fd8a6d55cc0cdd6b94886d7 :+1: